django_countries/fields.py

```diff
@@ -299,12 +302,15 @@ def pre_save(self, *args, **kwargs):
 
     def get_prep_value(self, value):
         "Returns field's value prepared for saving into a database."
-        if not self.multiple:
-            return country_to_text(value)
-        if isinstance(value, basestring):
-            return super(CharField, self).get_prep_value(value)
-        return ','.join(
-            country_to_text(code) for code in value if country_to_text(code))
+        if value is None:
+            return None
```

Comment:
Returning None doesn't seem like a good idea -- the field shouldn't be nullable anyway since it's a charfield. I think this is only required because you're changing the return value of multiple from a list to None